src/main/java/com/contentful/java/cma/ModuleAssets.java

```diff
@@ -204,9 +202,14 @@ public CMAAsset update(CMAAsset asset) {
     String spaceId = getSpaceIdOrThrow(asset, "asset");
     Integer version = getVersionOrThrow(asset, "update");
 
-    CMAAsset update = new CMAAsset();
-    update.setFields(asset.getFields());
-    return service.update(version, spaceId, assetId, update).toBlocking().first();
+    final CMASystem sys = asset.getSystem();
+    asset.setSystem(null);
```

Comment:
Otherwise it will get serialized to json and I don't want a sys field there in this case.